src/cli/input_notes.rs

```diff
@@ -280,6 +302,30 @@ where
     Ok(())
 }
 
+fn print_consumable_notes_summary<'a, I>(notes: I) -> Result<(), String>
+where
+    I: IntoIterator<Item = &'a ConsumableNote>,
+{
+    let mut table = create_dynamic_table(&["Note ID", "Account ID", "Relevance"]);
+
+    for note in notes {
+        for relevance in &note.1 {
+            table.add_row(vec![
+                note.0.id().inner().to_string(),
+                relevance.0.to_string(),
+                match relevance.1 {
+                    NoteRelevance::Always => "Always".to_string(),
+                    NoteRelevance::After(height) => format!("After block {}", height),
+                },
```

Comment:
nit: we should implement `Display` on `NoteRelevance` with a match similar to this one and here just use `relevance.1.to_string()`